toolkit/tools/pkg/imagecustomizerlib/testdata/consume-space.yaml

```diff
@@ -0,0 +1,3 @@
+SystemConfig:
```

Comment:
did you miss adding this to a test?

Reply:
We don't have functional tests yet. So, no. This file was added to make it easy to manually test this scenario in the future.